social-network-api/src/main/java/com/socialnetwork/api/service/EmailService.java

```diff
@@ -0,0 +1,36 @@
+package com.socialnetwork.api.service;
+
+import com.socialnetwork.api.model.ConfirmationToken;
+import com.socialnetwork.api.model.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.mail.javamail.MimeMessagePreparator;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class EmailService {
+  private final JavaMailSender javaMailSender;
+  private static final String EMAIL_ADDRESS_FROM = "java.test.email000@gmail.com";
+  private static final String CONFIRM_ACCOUNT_URL = "http://localhost:8080/registration/activate?token=";
```

Comment:
Please, move this to the application.properties too, because we can have not only localhost.